client/lib/sites-list/index.js

```diff
@@ -2,40 +2,40 @@
  * Internal Dependencies
  */
 import { action as InvitesActionTypes } from 'lib/invites/constants';
+import SitesList from './list';
+import PollerPool from 'lib/data-poller';
+import Dispatcher from 'dispatcher';
 
-var SitesList = require( './list' ),
-	PollerPool = require( 'lib/data-poller' ),
-	Dispatcher = require( 'dispatcher' ),
-	_sites;
+var sites;
 
 module.exports = function() {
-	if ( ! _sites ) {
-		_sites = new SitesList();
-		PollerPool.add( _sites, 'fetch' );
+	if ( ! sites ) {
+		sites = new SitesList();
+		PollerPool.add( sites, 'fetch' );
 
-		_sites.dispatchToken = Dispatcher.register( function( payload ) {
+		sites.dispatchToken = Dispatcher.register( function( payload ) {
 			var action = payload.action;
 			switch ( action.type ) {
 				case 'DISCONNECT_SITE':
 				case 'RECEIVE_DELETED_SITE':
-					_sites.removeSite( action.site );
+					sites.removeSite( action.site );
 					break;
 				case InvitesActionTypes.RECEIVE_INVITE_ACCEPTED_SUCCESS:
 					if ( [ 'follower', 'viewer' ].indexOf( action.invite.role ) === -1 ) {
-						_sites.sync( action.data );
+						sites.sync( action.data );
 					}
 					break;
 				case 'RECEIVE_DISCONNECTED_SITE':
-				case 'FETCH_SITES':
-					_sites.fetch(); // refetch the sites from .com
+				case 'FETCHsites':
```

Comment:
why did you change the capitalization of this action type? allcaps is the standard we're using everywhere else.